docker-compose.infra.yml

```
@@ -0,0 +1,19 @@
+services:
+  user-db:
+    image: postgres:15-alpine
+    container_name: user-db
+    ports:
+      - "15432:5432"
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=userdb
+    healthcheck:
+      test: [ "CMD-SHELL", "pg_isready -U postgres" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)